accumulator_updater/programs/accumulator_updater/src/lib.rs

```diff
@@ -18,27 +18,34 @@ declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 pub mod accumulator_updater {
     use super::*;
 
-    pub fn initialize(ctx: Context<Initialize>) -> Result<()> {
+    pub fn initialize(ctx: Context<Initialize>, authority: Pubkey) -> Result<()> {
```

Comment:
does this need to be permissioned?

Reply:
is there a way to have this permissioned other than hard-coding the initial authority? I wasn't planning on doing the initialization call through CPI. this could theoretically be a problem  since the current implementation doesn't support multiple whitelists (that's a future TODO) on the off-chance that someone randomly calls this initialize ix before we do but even in this scenario we would be able to re-deploy the program to a new address.

Reply:
I think It's ok to do it this way, unlikely someone initializes before us. Wormhole was setup this way also.